content/sensu-go/5.18/guides/generate-certificates.md

```diff
@@ -0,0 +1,195 @@
+
+# For systems with multiple hostnames and IP addresses, add each to the comma-delimited value of the ADDRESS variable
+export ADDRESS=localhost,127.0.0.1,10.0.0.1,backend-1
+export NAME=backend-1.example.com
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -profile="backend" -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" - | cfssljson -bare $NAME
+
+export ADDRESS=localhost,127.0.0.1,10.0.0.2,backend-2
+export NAME=backend-2.example.com
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -profile="backend" -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" - | cfssljson -bare $NAME
+
+export ADDRESS=localhost,127.0.0.1,10.0.0.3,backend-3
+export NAME=backend-3.example.com
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -profile="backend" -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" - | cfssljson -bare $NAME
+{{< /highlight >}}
+
+You should now have a set of files for each backend, where the `N` is the numeric element of the backend hostname:
+
+filename               | description                  | required on backend?|
+-----------------------|------------------------------|---------------------|
+`ca.pem`               | Trusted CA certificate.      | {{< check >}}       |
+`backend-N.pem`        | Backend server certificate.  | {{< check >}}       |
+`backend-N-key.pem`    | Backend server private key.  | {{< check >}}       |
+`backend-N.csr`        | Certificate signing request. |                     |
+
+Make sure to copy all backend PEM files and `ca-pem` to the corresponding backend system:
+
+{{< highlight shell >}}
+
+# Directory listing of /etc/sensu/pki/ on backend-1:
+/etc/sensu/pki/
+├── backend-1-key.pem
+├── backend-1.pem
+└── ca.pem
+{{< /highlight >}}
+
+These files should be accessible only by the `sensu` user.
+Use chown and chmod to make it so:
+
+{{< highlight shell >}}
+chown sensu /etc/sensu/pki/*.pem
+chmod 600 /etc/sensu/pki/*.pem
+{{< /highlight >}}
+
+### Generate agent certificate
+
+Now you will generate a certificate that agents can use to connect to the Sensu backend.
+Sensu's commerical distribution offers support for authenticating agents via TLS certificates instead of a username and password.
+
+For this certificate, you only need to specify a CN (here, `client`) &emdash; you don't need to specify an address.
+You will create the files `client-key.pem`, `client.csr`, and `client.pem`:
+
+{{< highlight shell >}}
+export NAME=client
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="" -profile=client - | cfssljson -bare $NAME
+{{< /highlight >}}
+
+## Next step: Secure Sensu
+
+Now that you have generated the required certificates, follow our [Secure Sensu][1] guide to make your Sensu installation production-ready.
```

Comment:
Add recap that you need to copy certain files to certain places. Installation doc should confront people with a choice to say "if you're going to set up a cluster, you need to go down a different path" and link them to the docs that cover that path.